package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "highlightjs-closure-templates",
+    "version": "0.0.1",
+    "description": "Google Closure Templates (soy)",
+    "main": "src/languages/soy.js",
+    "files": [
+        "src/languages/soy.js"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/highlightjs/highlightjs-closure-templates.git"
+    },
+    "keywords": [
+        "soy",
+	"closure",
+	"template",
+        "highlight.js",
+        "highlightjs",
+        "syntax-highlight",
+        "syntax"
+    ],
+    "author": "Chema Balsas",
+    "license": "BSD 3-clause",
+    "devDependencies": {
+    },
+    "dependencies": {
+        "highlight.js": "^10.4.1"
```

Comment:
10.4 is no longer supported, so I'd suggesting pinning against 11.

Reply:
Well at least Gerrit does not yet work with 11 (https://gerrit-review.googlesource.com/c/gerrit/+/322890).
So I'd at least like to keep it compatible with 10.

Reply:
Maybe we could specify `"^11.3.1 || 10.4.1"`?

Reply:
That sounds nice if that actually works.

Reply:
At least https://classic.yarnpkg.com/lang/en/docs/dependency-versions/#toc-unions describes that syntax and it seams to work with yarn in Gerrit.